images/go/image.yaml

```diff
@@ -9,3 +9,15 @@ versions:
       config: configs/latest.apko.yaml
       extractTagsFrom:
         package: go-1.20
+  - apko:
+      config: configs/1.19-dev.apko.yaml
+      extractTagsFrom:
+        package: go-1.19
+        prefix: dev-
+        exclude:
+          - 1
+  - apko:
+      config: configs/latest-dev.apko.yaml
+      extractTagsFrom:
+        prefix: dev-
```

Comment:
(not blocking)

This will produce `:dev-1.20.1-r0` which is also tagged `:latest-dev`?

We might want to standardize on whether `dev` comes first or last... should we have a `suffix:` option as well?

Reply:
Last. It should come last.